### DISCUSSION TITLE
Increase visibility of gpx-line?

### DISCUSSION THREAD

**Discussion by ERYpTION** [Helium314/SCEE#553] [Q&A]
I have managed to load a gpx-track that I need to follow on a survey. However, on the map the gpx-line is very faint (thin and light color). Especially using the map outside I have a very hard time seeing it. Is there an option to make it more visible (thickness and/or color)? Is this something that is determined by the gpx-file itself or the app?

Edit to add screenshot: 
![Screenshot_20240619-134007](https://github.com/Helium314/SCEE/assets/80313840/2d7272e8-5830-48c8-befa-17a2649b2533)

**Comment by Helium314** [comment]
It possibly depends on the theme. I don't have any problems on the black theme.
Making the line wider can easily look weird when the width is close to a path or road.

Anyway, this feature will have to be re-done when finally merging the MapLibre PR, so I don't think this is a good time for modifications here.
